### ISSUE TITLE
trying to use jamf launcher with fetch full installer

### ISSUE THREAD

**Issue by mrmte** [grahampugh/erase-install#488]
**Describe the bug**
trying to add a variable to the jamf launch script to get the full installer --fetch-full-installer rather than mist and upgrade macos but It errors out

**To Reproduce**
$4 = --current-user
$5 = --reinstall
$6 = --version=14.3.1
$7 = --check-power 
$8 =  --no-fs
$9 = --rebootdelay 300
$10 = --cleanup-after-use
$11= --fetch-full-installer
 
**Expected behavior**
to download the installer and continue the upgrade

**Code/log output**
Please supply the **full command used**, and if applicable, add **full output** from Terminal or from `/var/log/erase-install.log`. Don't post a partial log - I need the whole thing. Either upload the log as a file, or paste the output in a code block like this:

```
[erase-install-launcher] Starting script "--fetch-full-installer" --current-user --reinstall --version=14.3.1 --check-power --no-fs --rebootdelay 300 --cleanup-after-use
/Library/Application Support/JAMF/tmp/erase-install-launcher - Sonoma: line 96: --fetch-full-installer: command not found
[erase-install-launcher] Exit (127)
```

**Screenshots**
<img width="1007" alt="Screenshot 2024-02-21 at 13 23 35" src="https://github.com/grahampugh/erase-install/assets/2095960/5b25791b-0ade-4401-bbc3-ec98b8dfbd50">

**Environment (please complete the following information):**
 - OS version [e.g. 13.6]
 - erase-install version 13.1.0

**Additional context**

**Comment by grahampugh** [comment]
You can't use Parameter 11 - see the script for details on how to use it. 

As for the second comment, check if the softwareupdate command is working on your machine. If not, I would recommend removing the --fetch-full-installer option.

**Comment by mrmte** [comment]
Yeah I saw that and on the second attempt removed $11. Software update working no issues on my machine getting the full installer. I see you have the same bug logged you were able to reproduce.

**Comment by grahampugh** [comment]
You're right. I had forgotten about that, and yes it's the same issue. I've found the issue and it will be fixed in the next release.

**Comment by grahampugh** [comment]
https://github.com/grahampugh/erase-install/releases/tag/v33.0

**Comment by grahampugh** [comment]
Fixed in v33.0
